core/templates/dev/head/pages/creator_dashboard/creator-dashboard.mainpage.html

```diff
@@ -1125,7 +1125,8 @@ <h2>Suggestions to review</h2>
   <script src="/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js"></script>
 
   <script src="/templates/dev/head/domain/creator_dashboard/CreatorDashboardBackendApiService.js"></script>
-  <script src="/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js"></script>
+  <script src="/templates/dev/head/pages/creator_dashboard/creator-dashboard.module.js"></script>
```

Comment:
Are these scripts required here as they are already included in base.html and admin.html?

Reply:
I am not sure, (comment link https://github.com/oppia/oppia/pull/6524#discussion_r278119285). I will try to remove them and push the changes if those pass tests then ill keep them removed. 
And also the tests were failing when I did not include them in base or admin so I am not sure what is happening. I will try to remove them and then push changes and if it doesnt work then Ill revert it.